gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/mapper/ConditionMapper.java

```diff
@@ -107,21 +107,17 @@ public List<Condition> mapResources(RCMRMT030101UKEhrExtract ehrExtract, Patient
     }
 
     private Condition getCondition(Patient patient, List<Encounter> encounters, RCMRMT030101UKEhrComposition composition,
-                                   RCMRMT030101UKLinkSet linkSet, String practiseCode) {
+                                   RCMRMT030101UKLinkSet linkSet, String practiceCode) {
 
         String id = linkSet.getId().getRoot();
 
-        final Meta meta = confidentialityService.createMetaAndAddSecurityIfConfidentialityCodesPresent(
+        final var meta = confidentialityService.createMetaAndAddSecurityIfConfidentialityCodesPresent(
             META_PROFILE,
             linkSet.getConfidentialityCode(),
             composition.getConfidentialityCode()
         );
```

Comment:
I understand consistency is key, and this is perfectly fine! I just wanted to raise the point that I think having the type `Meta` isn't too bad instead of `var`. I like to use `var` on occasions where the class names are quite long! Also it gives me more insight on the type in which is returned!